### PR TITLE
Resprites some Cereal Boxes and adds recipes.

### PR DESCRIPTION
**Pull request by MitchTheUser**
[OBJECTS] [CATERING]

## About the PR 

Resprites the Honey Wonks and Tanh-O-Nys Cereal boxes, as well as gives the Flocked-Flakes Cirrial a unique sprite, as it previously shared the Shredded Syndies sprite. The granola bar has also been updated to be in 3/4th perspective. The sprite for Cookie Swirlies has also been moved up 1 pixel to line up with the other boxes.

The major feature of this pull request is the creation of recipes for creating every cereal box variant in an oven through different ingredients.

On 5 seconds of low heat, you can make the following new cereals with these ingredients (Honey Wonks already has a recipe):

- Cookie Swirlies: Sweet Dough + Chocolate Chips.
- Roach Puffs: Cookie Swirlies + Candy/Chocolate Bar.
- Tanh-O-Nys: Cookie Swirlies + Banana (With or without peel).
- Flocked-Flakes Cirrial: Cookie Swirlies + Odd Crystal (Flockdrone Brain).
- Shredded Syndies: Cookie Swirlies + Pure Telecrystal (From a Syndicate PDA, the joke is that you "purchase" the cereal).

Flocked-Flakes Cirrial previously contained Feather Fluid, as it was an admin only item. Since it now has a recipe to make the cereal, that has been replaced with a small amount of Coagulated Gnesis.

Food

## Why's this needed? 

Flocked-Flakes Cirrial had no unique sprite, and Honey Wonks/Tanh-O-Nys still had old cabinet perspective sprites which looked bad. Granola also wasn't in 3/4th perspective.

Most cereals are extremely hard to come by, only being found in certain maps in one place with only one b ox to go around, and a couple being unused entirely. This should allow more crewmates to enjoy the iconic breakfast food with way too much sugar.

### CODE REVIEW

File: code/modules/food_and_drink/snacks.dm [hunk 1]
```diff
@@ -807,12 +807,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 	prize = 0
 
 /obj/item/reagent_containers/food/snacks/cereal_box/cirrial
-	name = "cereal box -'Flocked-Flakes Cirrial'"
-	desc = "A bluey-green cereal that beeps gently at you, they're grrrrowing out of the box oh fuck!"
```
Comment (TDHooligan) [inline code/modules/food_and_drink/snacks.dm:811]: this is a frosted flakes/frosties reference, can probably stay in.

Comment (MitchTheUser) [reply code/modules/food_and_drink/snacks.dm:811]: Lol I didn't even realize that the name and description was one big Frosted Flakes reference. The description is back to how it was.

File: code/modules/food_and_drink/snacks.dm [hunk 2]
```diff
@@ -807,12 +807,12 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
...
-	icon_state = "cereal_box5"
-	initial_volume = 20
-	initial_reagents = list("feather_fluid"=10)
-	prize = 0
+    name = "cereal box -'Flocked-Flakes Cirrial'"
+    desc = "A bluey-green cereal that beeps gently at you, they're grrrrowing out of the box, oh fuck!"
+    icon_state = "cereal_box6"
+    initial_volume = 20
+    initial_reagents = list("flockdrone_fluid"=20)
```
Comment (TobleroneSwordfish) [inline code/modules/food_and_drink/snacks.dm:814]: This stuff evaporates if it's not in a gnesis container

Comment (MitchTheUser) [reply code/modules/food_and_drink/snacks.dm:814]: Removed the chemical from the cereal for now, may rework its contents in the future.

**Pull request merged**